scipy/stats/tests/test_continuous.py

```diff
@@ -1064,6 +1098,7 @@ def test_rv_generic(self, i, distdata):
                     3: {'pareto'},  # stats.pareto is just wrong
                     4: {'invgamma'}}  # tolerance issue
         skip_standardized = {'exponpow', 'ksone'}  # tolerances
+        skip_median = {'nhypergeom', 'yulesimon'}  # nan mismatch
```

Comment:
This new failures also seem to be due to forcing float endpoints for integer domains.

Reply:
That's fine. Let's study it separately. If it's a bad enough problem, add these distributions to the list of those that are currently incompatible with `make_distribution`.

Reply:
Locally, I need to add `'betanbinom', 'logser', 'zipf'`. Perhaps you're not seeing them because they're marked slow.

Reply:
And for all of these, it's because `X.cdf(X.support()[0]) > 0.5` - so it takes a really long time, and it seems to be the same old issue in `solve_bounded` with `_bracket_root` - there is not a valid bracket, so `_bracket_root` racks up iterations and eventually fails. Previously, I think these were producing the right answer, and now they are failing -  but I am not terribly concerned about that because it's something that needed to be fixed, whether they were returning the right answer or not. Let's skip the tests for now and address it in a follow-up.